### PR TITLE
Move UI components to separate files and add previews

### LINKED ISSUE #5: Chess piece view deserves separate file
It will also make possible to add couple of previews for it

### PR DESCRIPTION
**Pull request by Shiaulis**
# Desk View
image

# Board View
image

# Piece View
image

### CODE REVIEW

**Comment by Shiaulis** [comment]
This solves the ticket https://github.com/khaletska/chess/issues/5

**Comment by khaletska** [comment]
image

File: chess.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -12,7 +12,9 @@
 		B157A0E02C40365600810FCC /* ChessApp.swift in Sources */ = {isa = PBXBuildFile; fileRef = B157A0D92C40365500810FCC /* ChessApp.swift */; };
 		B157A0E12C40365600810FCC /* ChessModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = B157A0DA2C40365500810FCC /* ChessModel.swift */; };
 		B157A0E22C40365600810FCC /* ChessPiece.swift in Sources */ = {isa = PBXBuildFile; fileRef = B157A0DB2C40365500810FCC /* ChessPiece.swift */; };
-		B157A0E32C40365600810FCC /* ContentView.swift in Sources */ = {isa = PBXBuildFile; fileRef = B157A0DC2C40365500810FCC /* ContentView.swift */; };
+		B157A0E32C40365600810FCC /* DeskView.swift in Sources */ = {isa = PBXBuildFile; fileRef = B157A0DC2C40365500810FCC /* DeskView.swift */; };
```
Comment (khaletska) [inline chess.xcodeproj/project.pbxproj]: Does it make sense if we call it ChessGameView?

**Review by khaletska — CHANGES_REQUESTED** [review]
Only one comment

**Review by khaletska — APPROVED** [review]

**Pull request merged**